src/wasm-lib/kcl/src/std/extrude.rs

```diff
@@ -181,6 +181,48 @@ pub(crate) async fn do_post_extrude(
         vec![]
     };
 
+    for face_info in face_infos.iter() {
+        if face_info.cap == kittycad::types::ExtrusionFaceCapType::None
+            && face_info.face_id.is_some()
+            && face_info.curve_id.is_some()
+        {
+            match args
+                .batch_modeling_cmd(
+                    uuid::Uuid::new_v4(),
+                    kittycad::types::ModelingCmd::Solid3DGetOppositeEdge {
+                        edge_id: face_info.curve_id.unwrap(),
+                        object_id: sketch_group.id,
+                        face_id: face_info.face_id.unwrap_or_default(),
+                    },
```

Comment:
You check if both curve_id and face_id have `Some()` values but then unwrap here. I think it's safer if you pattern match them higher up and use their values here and below.

Reply:
ae401d286c98b7990994b37e41625920ab1a2344